### PR TITLE
www-app/gitea: Version bump to 1.17.0

### PR DESCRIPTION
**Pull request by eeyrjmr**
Signed-off-by: Jonathan Roadley-Battin <jon.roadleybattin@gmail.com>

### CODE REVIEW

**Review by stintel — CHANGES_REQUESTED** [review]
Aside from the keyword, gitea 1.17.0 built from this ebuild seems to run fine. Thanks!

File: www-apps/gitea/gitea-1.17.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+	dev-vcs/git"
+
+DOCS=(
+	custom/conf/app.example.ini CONTRIBUTING.md README.md
+)
+FILECAPS=(
+	-m 711 cap_net_bind_service+ep usr/bin/gitea
+)
+
+RESTRICT="test"
```
Comment (ajakk) [inline www-apps/gitea/gitea-1.17.0.ebuild:40]: Why? It seems like tests depend on nodejs, and with that installed the vast majority of tests seem to work for me.

**Review by ajakk — CHANGES_REQUESTED** [review]
Please squash your commits and fix the summary prefix (`www-app` -> `www-apps`). Probably should use pkgdev for that, so it's always done correctly.

**Comment by ajakk** [comment]
cc @thamognya

**Comment by stintel** [comment]
@eeyrjmr why did you close this?

**Pull request closed without merging**